ns1/datadog_checks/ns1/ns1_api_url.py

```diff
@@ -19,9 +19,9 @@
     # account plan limits
     "billing": "{apiendpoint}/v1/account/billataglance",
     # pulsar decisions account wide
-    "pulsar.decisions": "{apiendpoint}/v1/pulsar/query/decision/customer{query}",
+    "pulsar.decisions": "{apiendpoint}/v1/pulsar/query/decisions{query}",
```

Comment:
are these API changes backwards compatible?

Reply:
It's a new API endpoint, and code is changed to use it. Old will be deprecated eventually, don't have exact timeline for it

Reply:
Would existing customers be able to monitor from the new endpoint?

Reply:
It's new integration, there is no customers using it yet